classes/class-wc-qd-credit-manager.php

```diff
@@ -42,7 +42,7 @@ public function create_credit( $refund_id, $args ) {
 			'tag_list' => implode( ',', $invoice->tag_list ),
 			'processor' => 'woocommerce',
 			'processor_id' => $order->get_id(),
-			'payment_method' => self::get_payment_method($order->get_id()),
+			'payment_method' => self::get_payment_method($order),
 			'document_id' => $invoice_id,
```

Comment:
Shouldn't we add the `payment_processor` and `payment_processor_id` for credits as well?

Reply:
:+1: